schemas/v2.0.0.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  version: '2.0.0',
+  baseUrl: 'https://prod-1-dc-api-oopeix3r.deepcrawl.com',
```

Comment:
The schema only specifies the syntax for the API, not where that API lives. It can move from one base url to another, but as long as the version stays the same the schema should not require any changes.